cypress/fixtures/residents/3/helpRequests.json

```diff
@@ -283,7 +281,6 @@
         "WhenIsMedicinesDelivered": "",
         "RescheduledAt": "05:38",
         "RequestedDate": "2021-01-31T09:13:21.053Z",
-        "CaseNotes": "[{\"author\":\"Bilbo Baggins\",\"noteDate\":\"Thu, 7 Sep 2020 16:56:06 GMT\",\"note\":\"*** CREATED ***\"}]",
```

Comment:
So I think we'd still be getting case notes from this endpoint, not sure if we want to keep it close to the real thing. But also if we're not using it, it shouldn't be much of a difference